Source/TrackableEntities.EF.5/DbContextExtensions.cs

```diff
@@ -392,7 +392,7 @@ where typeof(ITrackable).IsAssignableFrom(prop.PropertyType)
             foreach (var item in items)
             {
                 // Avoid loading related entities of complext types, and avoid endless recursion
-                if (!visitationHelper.TryVisit(item) || IsComplexType(context, item.GetType())) continue;
+                if (IsComplexType(context, item.GetType()) || !visitationHelper.TryVisit(item)) continue;
```

Comment:
No problem for this PR, but in the future try to add the failing test in one comment, then add the fix to another commit.  That way, I can do a soft reset to the commit with the failing test to see it fail, and then reset to the fit commit to see it pass.  Just a bit easier that way.  But don't worry about it this time.